ansible/libvirt-vm-single.yml

```diff
@@ -71,10 +71,9 @@
 - name: Wait for SSH access to the VM
   local_action:
     module: wait_for
-    host: "{{ hostvars[vm_host].vm_ip }}"
+    host: "{{ hostvars[vm_host].vm_configdrive_network_device_list[0].address }}"
```

Comment:
I think it was more flexible before. You could set the interfaces in any order then decide how to access the VM using vm_ip.

Reply:
I suppose we could always:
```
vm_ip: "{{ hostvars[vm_host].vm_configdrive_network_device_list[0].address }}"